### PR TITLE
Make bodhi distinguish about updates going to testing vs stable when asking greenwave

### PR DESCRIPTION
**Pull request by pypingou**
This will allow us and greenwave to apply different policies for
depending on which repo is update is going (updates-testing vs updates)
which is required to integrate some of the more useful taskotron tests.

Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

**Comment by ralphbean** [comment]
:+1:

Can you update the greenwave policy in ansible too to add a new decision context?

**Comment by pypingou** [comment]
Sure thing though I was thinking we may want to discuss some more what we want for each context

File: bodhi/server/scripts/check_policies.py [hunk 1]
```diff
@@ -41,9 +41,12 @@ def check():
         .filter(models.Update.status.in_(
                 [models.UpdateStatus.pending, models.UpdateStatus.testing]))
     for update in updates:
+        decision_context = u'bodhi_update_push_testing'
+        if update.status == models.UpdateStatus.testing:
+            decision_context = u'bodhi_update_push_stable'
```
Comment (bowlofeggs) [inline bodhi/server/scripts/check_policies.py]: I think this code is right, but it is also confusing initially that we'd mix "testing" and "stable" like this. Can we add a comment explaining why we want to check the stable context when it is in testing? Maybe something like this:

```
# We want to know a decision about pending updates going to testing, and about testing updates going to stable.
```

File: bodhi/tests/server/scripts/test_check_policies.py [hunk 2]
```diff
@@ -59,6 +59,35 @@ def test_policies_satisfied(self):
                                                expected_query)
 
     @patch.dict(config, [('greenwave_api_url', 'http://domain.local')])
+    def test_policies_pending_satisfied(self):
+        """Assert correct behavior when the policies enforced by Greenwave are satisfied"""
```
Comment (bowlofeggs) [inline bodhi/tests/server/scripts/test_check_policies.py]: This comment doesn't quite capture the purpose of the test as I understand it - can we reword it to state that we are ensuring that pending updates use the ```bodhi_update_push_testing``` decision context? That way it'll be clearer what this test is for later.

**Review by bowlofeggs — APPROVED** [review]
Code looks good, I just have a couple of comment changes I'd like to see to clarify things. Merge when ready (no need for re-review)!

**Comment by pypingou** [comment]
Both comment adjusted :)

Will let jenkins run and merge afterward

**Review by bowlofeggs — APPROVED** [review]
Thanks!
